pages.de/common/az-login.md

```diff
@@ -0,0 +1,25 @@
+# az login
+
+> Melden Sie sich bei Azure an.
```

Comment:
We always try to avoid personal pronouns and write descriptions in imperative mood, but this is a tough case. I think we have to make an exception here.